.Lib9c.Tests/Model/RankingSimulatorV1Test.cs

```diff
@@ -158,14 +158,14 @@ public void ConstructorWithCostume()
             );
 
             var player = simulator.Player;
-            Assert.Equal(row.Stat, player.Stats.OptionalStats.ATK);
+            Assert.Equal(row.Stat, player.Stats.CostumeStats.ATK);
```

Comment:
I think this means `Optional` type of things (like former costume) will not be used in the future, right?

Reply:
yes, the name of OptionalStats was changed because only costumes were being used. This change is separate from the change of stat calculation.